src/component-library/VaultCard/VaultCard.tsx

```diff
@@ -43,19 +45,17 @@ const VaultCard = ({
         <DlItem>
           {/* TODO: these headings will also be moved to the dictionary */}
           <dt>Pending requests</dt>
-          <dd>{pendingRequests}</dd>
+          <dd>{formatNumber(pendingRequests)}</dd>
```

Comment:
hi @crypto-engineer - do you think we should be handling formatting higher up, rather than importing the utility into the application?

Reply:
Yes for sure. Sorry I did not properly thought about it.

Reply:
Done!